### PR TITLE
front: first implementation e2E with smoke test select infra on grille horai…

### PR DESCRIPTION
**Pull request by Math-R**

### CODE REVIEW

File: front/.github/workflows/playwright.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    - name: Install Playwright Browsers
+      run: yarn playwright install --with-deps
+    - name: Run Playwright tests
+      run: yarn playwright test
+    - uses: actions/upload-artifact@v3
+      if: always()
+      with:
+        name: playwright-report
+        path: playwright-report/
+        retention-days: 30
```
Comment (Khoyo) [inline front/.github/workflows/playwright.yml]: Shouldn't this be part of `.github/workflows/front.yml` , not `front/.github/workflows/playwright.yml`?

Comment (flomonster) [reply front/.github/workflows/playwright.yml]: It was moved to integration tests.

**Review by flomonster — CHANGES_REQUESTED** [review]
Few things must be done before merging:

- Remove deadcode such as configuration example (I think it was auto generated)
- Reduce timeouts (we shouldn't wait 30s for a test to complete)
- Rebase and fix the failing test

File: .github/workflows/integration_tests.yml [hunk 2]
```diff
@@ -23,24 +26,43 @@ jobs:
...
 
       - name: Run tests
         run: "python3 tests/run_integration_tests.py"
+
+      - name: Run Playwright tests
+        run: yarn --cwd front playwright test
+      - uses: actions/upload-artifact@v3
+        with:
+          name: playwright-report
+          path: front/playwright-report/
```
Comment (flomonster) [inline .github/workflows/integration_tests.yml]: Missing line feed.

**Review by flomonster — CHANGES_REQUESTED** [review]

**Review by flomonster — APPROVED** [review]

**Review by nicolaswurtz — APPROVED** [review]

**Pull request merged**